src/developer/core/network-api.md

```diff
@@ -162,7 +162,8 @@ Messages can have any topic chosen by the user. See topics below for a list of m
 
 ### Pupil and Gaze Messages
 
-Pupil data is sent from the eye0 and eye1 process with the topic `pupil.0` or `pupil.1`.
+Pupil data is sent from the eye0 and eye1 process with the topic format `pupil.<EYE_ID>.<PUPIL_DETECTOR_IDENTIFIER>`, where `EYE_ID` is `0` or `1` for eye0 and eye1 respectively, and `PUPIL_DETECTOR_IDENTIFIER` is a string that uniquely identifies the pupil detector plugin that produced the message. In the case of the built-in pupil detectors, the identifier corresponds to `2d` and `3d` respectively.
```

Comment:
@papr actual example of `pupil.0.2d` or `pupil.0.3d` relevant here?

Reply:
I think it would make sense to list the built-in topics. @romanroibu please add the 4 possibilities as examples